<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeLoginDialog</class>
 <widget class="QDialog" name="ChangeLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="oldLabel">
     <property name="text">
      <string>Old login</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="loginLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="newLabel">
     <property name="text">
      <string>New login</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="repeatLabel">
     <property name="text">
      <string>Repeat new login</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="changeButton">
     <property name="text">
      <string>Change login</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="newEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="repeatEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
